clusterloader2/pkg/measurement/common/slos/slo_measurement.go

```diff
@@ -68,6 +72,23 @@ func (s sloMeasurement) Execute(config *measurement.Config) ([]measurement.Summa
 	return s.summaries, nil
 }
 
+func getMeasurementConfig(config *measurement.Config, measurement_name string) (*measurement.Config) {
+	measurementConfig := *config
+	measurementConfig.Params = util.CloneMap(config.Params)
+
+	overrides, err := util.GetMap(config.Params, "overrides")
+	if err == nil {
```

Comment:
You shouldn't silently ignoring the error.

Reply:
In other words, this function should return (Config, error)

Reply:
Done.